src/_pytest/pytester.py

```diff
@@ -1587,6 +1588,6 @@ def _fail(self, msg: str) -> None:
         self._log_output = []
         pytest.fail(log_text)
 
-    def str(self) -> str:
+    def string(self) -> str:
```

Comment:
This is a breaking change, we should keep the original name

Reply:
thanks, I didn't realize. Just changed it back